### ISSUE TITLE
Fix timezone issues

### ISSUE THREAD

**Issue by JanMalch** [JanMalch/kino#134]
z.B. bei der Registrierung wird der Geburtstag im Endeffekt auf den Tag davor gesetzt (GMT+2 etc.).

Die JSON vom Server enthält bereits das falsche Datum

**Comment by JanMalch** [comment]
Können naiv beim Speichern des Geburtstags auf das Datum einfach 23h 59m 59s draufklatschen. Oder 12h oder so.

**Comment by kollb** [comment]
Wie wärs wenn wir die Problematik mit GMT +2 einfach umgehen, indem wir einfach nur GMT benutzen? Da wo wir +2 haben, einfach rausnehmen.

**Comment by JanMalch** [comment]
Ja, das müssten wir aber irgendwie serverseitig machen, dass er das richtig empfängt. wenn man im frontend halt nur den String `'2019-06-24'` absendet, hab ich keine Information über Zeitzone und der nimmt GMT+0 an.

**Comment by JanMalch** [comment]
See [ToDo].(https://github.com/JanMalch/kino/blob/master/src/main/java/io/github/janmalch/kino/control/myaccount/EditMyAccountControl.java#L81-L83)
